rtmp-client/src/main/java/net/butterflytv/rtmp_client/RtmpClient.java

```diff
@@ -48,9 +54,40 @@ public RtmpIOException(int errorCode) {
 
     }
 
+    /**
+     *  Sets the socket's send timeout value
+     * @param sendTimeoutInMs
+     * The send timeout value for the rtmp socket in milliseconds.
+     * Parameter expects a non-zero positive integer and will reset timeout to the default value
+     * (10000 ms) if zero or a negative integer is passed.
+     *  */
+    public void setSendTimeoutInMs(int sendTimeoutInMs) {
```

Comment:
nit: Can we call these methods `setSendTimeout` and `setReceiveTimeout`? The names of the arguments are sufficient to make the unit clear, so there's no need to also document this in the method name.

Reply:
Done